src/state/slices/limitOrderSlice/selectors.ts

```diff
@@ -36,17 +35,17 @@ export const selectActiveQuoteExpirationTimestamp = createSelector(
 )
 
 export const selectActiveQuoteSellAsset = (state: ReduxState) => {
-  const sellAssetId = state.limitOrderSlice.activeQuote?.params.sellAssetId
+  const sellAssetId = state.limitOrder.activeQuote?.params.sellAssetId
   return sellAssetId ? selectAssetById(state, sellAssetId) : undefined
 }
 
 export const selectActiveQuoteBuyAsset = (state: ReduxState) => {
-  const buyAssetId = state.limitOrderSlice.activeQuote?.params.buyAssetId
+  const buyAssetId = state.limitOrder.activeQuote?.params.buyAssetId
   return buyAssetId ? selectAssetById(state, buyAssetId) : undefined
 }
 
 export const selectActiveQuoteFeeAsset = (state: ReduxState) => {
-  const sellAssetId = state.limitOrderSlice.activeQuote?.params.sellAssetId
+  const sellAssetId = state.limitOrder.activeQuote?.params.sellAssetId
   return sellAssetId ? selectFeeAssetById(state, sellAssetId) : undefined
 }
```

Comment:
should these be updated to `createSelector` to leverage any memoization?

Reply:
Ohh 100%. These are super bad, selecting the whole state and running selectors inside it is hell 💩, it basically equals to being reactive on... the whole store. That would probably explain why limit is so flaky.

Reply:
d5a2f1c5e4 no dependency on global `state` anymore 🎉